content/en/developers/metrics/types.md

```diff
@@ -2,298 +2,327 @@
 
-| Metric Name                                    | Value  | Datadog Metric Type |
-|------------------------------------------------|--------|---------------------|
+| Metric Name                                    | Value  | Datadog In-app Type |
+| ---------------------------------------------- | ------ | ------------------- |
 | `request.response_time.histogram.avg`          | `1,88` | GAUGE               |
 | `request.response_time.histogram.count`        | `8`    | RATE                |
 | `request.response_time.histogram.median`       | `2`    | GAUGE               |
 | `request.response_time.histogram.95percentile` | `3`    | GAUGE               |
 | `request.response_time.histogram.max`          | `3`    | GAUGE               |
 
-**Note**:
-
-* Configure which aggregation you want to send to Datadog with the `histogram_aggregates` parameter in your [datadog.yaml configuration file][2]. By default, only `max`, `median`, `avg`, and `count` aggregations are sent out to Datadog. `sum` and `min` are available for addition.
-* Configure which percentile aggregation you want to send to Datadog with the `histogram_percentiles` parameter in your [datadog.yaml configuration file][2]. By default, only the `95pc` percentile is sent out to Datadog.
-
-Discover how to submit histogram metrics:
-
-* [With a custom Agent Check][3]
-* [With DogStatsD][4]
-
-[1]: https://github.com/etsy/statsd/blob/master/docs/metric_types.md#timing
-[2]: /agent/guide/agent-configuration-files/#agent-main-configuration-file
-[3]: /developers/metrics/agent_metrics_submission/?tab=histogram
-[4]: /developers/metrics/dogstatsd_metrics_submission/#histogram
 {{% /tab %}}
 {{% tab "DISTRIBUTION" %}}
 
-**A `DISTRIBUTION` is a metric submission type that allows you to aggregate values sent from multiple hosts during a flush interval to measure statistical distributions across your entire infrastructure.** `DISTRIBUTION` metrics are designed to instrument logical objects, like services, independently from the underlying hosts.
-
-Unlike the `HISTOGRAM` metric type, which aggregates on the Agent side during the flush interval, a `DISTRIBUTION` metric sends all the raw data during a flush interval to Datadog, and aggregations occur server-side. Because the underlying data structure represents raw, unaggregated data, distributions provide two major features:
-
-* Calculation of percentile aggregations
-* Customization of tagging
+Suppose you’re submitting a DISTRIBUTION metric, `request.response_time.distribution`, from two webservers: `webserver:web_1` and `webserver:web_2`. Suppose in one time interval, webserver:web_1 reports the metric with the values `[1,1,1,2,2,2,3,3]`, and `webserver:web_2` reports the same metric with the values `[1,1,2]`. Over this time interval, the following 5 aggregations will represent the global statistical distribution of all values collected from both webservers:
```

Comment:
Suppose you’re submitting a DISTRIBUTION metric, `request.response_time.distribution`, from two webservers: `webserver:web_1` and `webserver:web_2`. Suppose in a given flush time interval, webserver:web_1 reports the metric with the values `[1,1,1,2,2,2,3,3]`, and `webserver:web_2` reports the same metric with the values `[1,1,2]`. Over this time interval, the following 5 aggregations will represent the global statistical distribution of all values collected from both webservers: